man/networkd.conf.xml

```diff
@@ -61,6 +61,13 @@
         <listitem><para>Specifies the time interval to calculate the traffic speed of each interface.
         If <varname>SpeedMeter=no</varname>, the value is ignored. Defaults to 10sec.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>DropForeignRoutes=</varname></term>
+        <listitem><para>A boolean. When true <command>systemd-networkd</command> will not store any foreign routes
+        in it's memory. Defaults to false.</para></listitem>
+      </varlistentry>
```

Comment:
What functionality is lost by not remembering these foreign routes? in other words, why does systemd-networkd have to store these routes in the first place?

Reply:
I am pretty sure you missed the discussion. And I formerly think you should remember or Google it. Anyway why should networkd allow to an one control the interface and situation like this keep on happening like keepalived. Genuine cases. Can you just see the history

Reply:
Sure. What I was implicitly suggesting is that the documentation for the new option should mention what functionality is lost if the option is enabled, and/or whether it's incompatible with any other config directives.

Reply:
I like that. We can say keepconfiguration won't be in same way since this happens very early.